.github/workflows/ci.yml

```diff
@@ -123,36 +122,35 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: Setup uv
-      uses: astral-sh/setup-uv@v3
+    - uses: astral-sh/setup-uv@v3
       with:
         enable-cache: true
 
-    - name: install deps
-      run: uv sync
+    - name: Install dependencies
+      run: uv sync --extra timezone
 
     - run: 'uv run python -c "import pydantic.version; print(pydantic.version.version_info())"'
 
     - run: mkdir coverage
 
-    - name: test without deps
+    - name: Test without email-validator
```

Comment:
Previously, this was a bit weird because even though we did not install any extra, `tzdata` (which is installed through the `timezone` extra) was still included in the `dev` dependency group. I've tried changing the pytest skip marker for these tests to check for the presence of the `tzdata` library but it's tricky as at the module level, some `ZoneInfo` instances are created so it still fails.

In the future, if we include new extras, we should change the name to `Test only with 'timezone' extra`

Reply:
This feels cleaner to me than the way we had it before, I wasn't a fan of having `tzdata` in `dev`. Thanks!